protocols/isis/server/lsdb.go

```diff
@@ -0,0 +1,4 @@
+package server
+
+type lsdb struct {
```

Comment:
this struct is not used yet, if it is part of already removed code, it should be removed, too